### PR TITLE
Possible fix issue 4611: Don't write to database before it is created

### PR DESCRIPTION
**Pull request by annando**
I'm not sure if this really fixes issue https://github.com/friendica/friendica/issues/4611 but we really shouldn't write into database tables before we created them.

### CODE REVIEW

**Comment by tobiasd** [comment]
I'll check if that solves the issue.

File: src/Database/DBStructure.php [hunk 1]
```diff
@@ -203,8 +203,8 @@ private static function printUpdateError($message) {
 	 * @param array $definition An array of the definition tables
 	 * @return string Empty string if the update is successful, error messages otherwise
 	 */
-	public static function update($verbose, $action, array $tables = null, array $definition = null) {
-		if ($action) {
+	public static function update($verbose, $action, $install = false, array $tables = null, array $definition = null) {
```
Comment (rabuzarus) [inline src/Database/DBStructure.php]: Please add the `$install` parameter to the doxygen header

**Comment by tobiasd** [comment]
look good :-)

File: src/Database/DBStructure.php (hunk 1 above)
Comment (annando) [reply src/Database/DBStructure.php]: Oops

**Pull request merged**